swagger_parser/README.md

```diff
@@ -53,9 +53,6 @@ swagger_parser:
   # Sets the OpenApi schema path directory for api definition.
   schema_path: schemas/openapi.json
 
-  # Sets the url of the OpenApi schema
-  schema_url: https://petstore.swagger.io/v2/swagger.json
-
```

Comment:
Why did you delete it?

Reply:
https://github.com/Carapacik/swagger_parser/pull/126